app/views/layouts/application.html.haml

```diff
@@ -7,4 +7,11 @@
     = stylesheet_link_tag    'application', media: 'all', data: {'turbolinks-track' => 'reload'}
     = javascript_include_tag 'application', data: {'turbolinks-track' => 'reload'}
   %body
+    #categories
+      %ul
+        - @categories.each do |category|
+          %li= link_to category.name, category
```

Comment:
Why is this in the application.html.haml?

Reply:
Since we want the products to be displayed under each category site, and available across each view in a header, we added the link to each category here.